test/sqllogictest/string.slt

```diff
@@ -31,9 +31,6 @@ column1
 statement ok
 CREATE TABLE asciitest (strcol CHAR(15), vccol VARCHAR(15))
 
-# TODO: materialize#589 Add two tests:
```

Comment:
The two points below did not get done. This should stay, but the issue ref can be removed.

Reply:
Shall we create a new issue to clean this up and reference that?

Reply:
Should be simple enough to do it inline here, I just didn‘t understand what tests there were supposed to be.

Reply:
@sploiselle Do you understand what was kind of test was missing here?

Reply:
Yeah, this just wanted to add some "edge cases" to these tests. The state of MZ's string support when these tests were written was less sophisticated than it is now. Just to be a good citizen, I've added the tests in #22886 though I don't think this meaningfully improves our test coverage.